### PR TITLE
(fix) tally  alignment issue in radiology tabs

### PR DESCRIPTION
**Pull request by FelixKiprotich350**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary
fixed the tabs count alignment with the number of items in that tab. Unlike before tab count was not matching the number of items in the tabPanel

## Screenshots
Screenshot from 2024-07-19 14-20-54
Screenshot from 2024-07-19 14-21-15

### CODE REVIEW

File: src/summary-tiles/radiology-summary.resource.tsx [hunk 1]
```diff
@@ -32,25 +32,57 @@ export function useProcedureOrderStats(fulfillerStatus: string) {
...
-      order.concept.conceptClass.uuid === "8caa332c-efe4-4025-8b18-3398328e1323"
-    ) {
-      return order;
+    if (fulfillerStatus === "") {
+      return (
+        order.fulfillerStatus === null &&
+        order.dateStopped === null &&
+        order.action === "NEW" &&
+        order.concept.conceptClass.uuid ===
+          "8caa332c-efe4-4025-8b18-3398328e1323"
```
Comment (makombe) [inline src/summary-tiles/radiology-summary.resource.tsx:43]: Can we move conceptClassUuid to constant file  and add something like`export const ImagingConceptClass_UUID ="8caa332c-efe4-4025-8b18-3398328e1323";` then you can references it here

**Review by makombe — COMMENTED** [review]
Thanks @FelixKiprotich350 LGTM

**Comment by FelixKiprotich350** [comment]
Sure this is a concern that I had but I can make the change now and push to
the same pr

File: src/summary-tiles/radiology-summary.resource.tsx (hunk 1 above)
Comment (FelixKiprotich350) [reply src/summary-tiles/radiology-summary.resource.tsx]: @makombe Done

**Comment by makombe** [comment]
@FelixKiprotich350 Fix the lint and we are good to go. Thanks

**Comment by FelixKiprotich350** [comment]
@makombe Good to go. lint issues fixed

**Review by makombe — APPROVED** [review]

**Comment by ojwanganto** [comment]
Thanks, @FelixKiprotich350 and @makombe

**Pull request merged**